style.css

```diff
@@ -392,61 +391,90 @@ body {
   padding-top: 10px;
 }
 
-.image-container {
+.partners1 {
+  background-color: #272a31;
+  color: #dedede;
+  padding-bottom: 20px;
+}
+
+.partners1 h3 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
   position: relative;
-  width: 400px;
 }
 
-.past-events {
-  background-color: #ccc;
+.partners1 h3::after {
+  content: "";
+  position: absolute;
+  bottom: -8px;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 30px;
+  height: 2px;
+  background-color: #ec5242;
+}
+
+.partners1 img {
+  width: 100px;
 }
 
-.past-image {
-  /* display: block; */
-  width: 80%;
+.partners-card1 {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(120px, 1fr));
+  grid-gap: 5px;
+  padding-top: 10px;
+}
 
-  /* padding: 40px; */
+#past-ev {
+  background-color: #e7e7e7;
+  padding: 1em 1em;
+}
+
+.gallery-img {
   display: flex;
   flex-direction: column;
-  gap: 1px;
+  gap: 10px;
   align-items: center;
   margin-top: 1em;
 }
 
-.image-overlay {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 80%;
-  height: 100%;
-  background: #ec53428c;
-  color: #dedede;
+.con {
   display: flex;
-  flex-direction: column;
   justify-content: center;
+  flex-direction: column;
   align-items: center;
-  opacity: 0;
-  transition: opacity 0.3s;
+  color: #fff;
 }
 
-.image-overlay:hover {
-  opacity: 1;
+.con h2 {
+  font-size: 3rem;
 }
 
-.image-title {
-  font-size: 2em;
-  font-weight: bold;
+.conf1 {
+  background-image: linear-gradient(#f5331dbf, #f2331ea2), url("images/pcm.jpg");
+  background-size: cover;
+  height: 300px;
+  width: 100%;
+  cursor: pointer;
+  transition: eall 0.5s;
 }
 
-.image-description {
-  font-size: 1.25em;
-  margin-top: 0.25em;
+.conf1:hover {
+  background-image: linear-gradient(#f5331d00, #f2331e00), url("images/pcm.jpg");
 }
 
 #footer {
   display: flex;
   justify-content: space-between;
-  background-color: #dedede;
+  background-color: #f6f4f4;
+  padding: 0;
+}
+
+#footer2 {
+  display: flex;
+  justify-content: space-between;
+  background-color: #f6f4f4;
   padding: 0;
 }
```

Comment:
Great job @Eccb7 🤝

- [ ] As stated by the previous reviewer, kindly remove the footer from your mobile view of the home page.
- Your design
![Your mobile footer](https://github.com/Eccb7/Capstone_Project-1/assets/63917725/c7cc6660-fa0d-4015-84d2-59e30da6663f)
- Given design
![Given mobile footer](https://github.com/Eccb7/Capstone_Project-1/assets/63917725/6a60f15d-cc92-49ce-8484-124ba3d3d173)